packages/cli/cli/src/cli.ts

```diff
@@ -644,41 +645,48 @@ function addUpgradeCommand({
     );
 }
 
-// function addUpgradeGeneratorCommand(cli: Argv<GlobalCliOptions>, cliContext: CliContext) {
-//     cli.command(
-//         "generator upgrade",
-//         `Upgrades the specified generator in ${GENERATORS_CONFIGURATION_FILENAME} to the specified version, or latest if one is not specified.`,
-//         (yargs) =>
-//             yargs
-//                 .option("generator", {
-//                     string: true,
-//                     description: "The type of generator to upgrade, ex: `fern-typescript-node-sdk`."
-//                 })
-//                 .option("group", {
-//                     string: true,
-//                     description:
-//                         "The group in which the generator is located, if group is not specified, the all generators of the specified type will be upgraded."
-//                 })
-//                 .option("version", {
-//                     string: true,
-//                     description: "The version to upgrade to. Defaults to the latest release."
-//                 }),
-//         async (argv) => {
-//             cliContext.instrumentPostHogEvent({
-//                 command: "fern generator upgrade",
-//                 properties: {
-//                     generator: argv.generator,
-//                     version: argv.version
-//                 }
-//             });
-//             await upgradeGenerator({
-//                 cliContext,
-//                 generator: argv.generator,
-//                 targetVersion: argv.version
-//             });
-//         }
-//     );
-// }
+function addUpgradeGeneratorCommand(cli: Argv<GlobalCliOptions>, cliContext: CliContext) {
+    cli.command(
+        "generator upgrade",
```

Comment:
nit: Not actionable here, but is this the best way to handle sub-commands with `yargs`? I'm curious what the user will see if they type `fern generator --help` in order to discover what sub-commands they have available to them. It's unclear from a quick search. Seems like it might be tucked away in [here](https://github.com/yargs/yargs/blob/main/docs/advanced.md).

Not at all important for the first iteration, so feel free to ignore.